### DISCUSSION TITLE
Is there any way to attach a stream to a conference.?

### DISCUSSION THREAD

**Discussion by Jhoubert** [ant-media/Ant-Media-Server#5107] [General]
Hi all! I'm building an interface based on [conference-call-application](https://github.com/ant-media/conference-call-application/issues/92) . But I want to use OBS to stream to a conference among the participants. Is there any way to do that without using a "virtual camera" or "sharing the screen" ?.

That would be something like:

### Conference 01:
 - Person 01
 - Person 02
 - STREAM FROM OBS
 - Person 03

**NOTE: I understand that probably is not possible right now, but any idea to follow would be a good point to start with and will be really appreciated!** 

_EDIT: Haven't digged into the server's repo yet. so, pointing any package/module to start reading to would be awesome!_

Thanks in advance!

**Comment by Jhoubert** [comment]
Just noticed that in `ant-media-server` the package `io.antmedia.enterprise` (I'm just guessing the name) is not OpenSource.
![image](https://github.com/ant-media/Ant-Media-Server/assets/39770320/e99c6a5f-d178-4aaf-942a-d102a1b43989)

**Comment by yashtandon113** [comment]
@Jhoubert 
Thanks for your question.
Yes, you can add your OBS RTMP stream to the conference by using the add sub-track Rest API or the add particular stream to the room Rest API request. Check these out:
https://antmedia.io/rest/#/BroadcastRestService/addStreamToTheRoom
https://antmedia.io/rest/#/BroadcastRestService/deleteStreamFromTheRoom
https://antmedia.io/rest/#/BroadcastRestService/addSubTrack

**Reply by Jhoubert** [reply]
That's exactly what I was looking for! Thanks you!
